src/NzbDrone.Core/DecisionEngine/DownloadDecisionPriorizationService.cs

```diff
@@ -30,15 +31,26 @@ public List<DownloadDecision> PrioritizeDecisions(List<DownloadDecision> decisio
                                 {
                                     var downloadDecisions = d.ToList();
                                     var series = downloadDecisions.First().RemoteEpisode.Series;
-
-                                    return downloadDecisions
-                                        .OrderByDescending(c => c.RemoteEpisode.ParsedEpisodeInfo.Quality, new QualityModelComparer(series.Profile))
-                                        .ThenBy(c => c.RemoteEpisode.Episodes.Select(e => e.EpisodeNumber).MinOrDefault())
-                                        .ThenBy(c => PrioritizeDownloadProtocol(series, c.RemoteEpisode.Release.DownloadProtocol))
-                                        .ThenByDescending(c => c.RemoteEpisode.Episodes.Count)
-                                        .ThenBy(c => c.RemoteEpisode.Release.Size.Round(200.Megabytes()) / Math.Max(1, c.RemoteEpisode.Episodes.Count))
-                                        .ThenByDescending(c => TorrentInfo.GetSeeders(c.RemoteEpisode.Release))
-                                        .ThenBy(c => c.RemoteEpisode.Release.Age);
+                                    if (series.Profile.Value.LanguageOverQuality)
```

Comment:
Too much duplication here, only the first lines are changing, but 6 other lines are being copied, since we're dealing with IEnumerables here nothing will be evaluated until the end, so we can order by the first two clauses (in the appropriate order) then add the other six.